data/mapcss-mapping.csv

```diff
@@ -427,8 +427,8 @@ highway|cycleway|permissive;[highway=cycleway][access=permissive];;name;int_name
 highway|unclassified|area;[highway=unclassified][area?];;name;int_name;354;
 natural|volcano;355;
 deprecated:amenity|parking|underground|fee:01.2020;[amenity=parking][location=underground][fee?],[amenity=parking][parking=underground][fee?];x;name;int_name;356;amenity|parking|underground
-amenity|parking|underground;[amenity=parking][location=underground],[amenity=parking][parking=underground];;name;int_name;357;
```

Comment:
hmm, why did you remove the `[amenity=parking][location=underground]` mapping?

Reply:
oohhh thank you! this is why the tests are failing 👍